sof_wrapper/api/fhir.py

```diff
@@ -19,6 +19,42 @@ def collate_results(*result_sets):
     return results
 
 
+def add_cds_extensions(med):
+    """Add FHIR extensions as necessary to support frontend CDS"""
+
+    expected_supply_duration = med.get('dispenseRequest', {}).get('expectedSupplyDuration', {}).get('value')
+    quantity = med.get('dispenseRequest', {}).get('quantity', {}).get('value')
+    if not expected_supply_duration or not quantity:
+        return med
+
+    # TODO: reduce frequency to lowest terms (fractions.gcd), if necessary
+    dosage_instruction = [{
+        'timing': {
+            "repeat": {
+                "frequency": quantity,
+                "period": expected_supply_duration,
+                "periodUnit": "d"
+            }
+        },
+        "doseAndRate": [
+            # dosage per day
+            {"doseQuantity": {"value": quantity/expected_supply_duration}}
+        ]
+    }]
+
+    annotated_med = med.copy()
+    annotated_med['dispenseRequest'].setdefault('dosageInstruction', dosage_instruction)
+
+    return annotated_med
+
+
+def annotate_meds(med_bundle):
+    annotated_bundle = {'resourceType': 'Bundle', 'entry': [], 'total': med_bundle['total']}
```

Comment:
bundles occasionally have additional meta data - as we control the source, this may not be an issue, but why not simply replace the 'entry'?

Reply:
Good point! I'll copy and unset the `entry` key